Add cash and check handling policies (#144)
RoboJackets/business-process-manual: corporate/finance/cash-handling.rst

:og:description: While we prefer electronic payments, RoboJackets accepts cash payments from students for dues and trip fees. Payments must be recorded in Apiary at the time the payment is accepted from the student, and Apiary will send an email receipt to the student.

Cash handling policy
====================

.. vale Google.Passive = NO
.. vale Google.We = NO
.. vale Google.Will = NO
.. vale write-good.E-Prime = NO
.. vale write-good.Passive = NO

While we prefer electronic payments, RoboJackets accepts cash payments from students for dues and trip fees.

> Payments must be `recorded in Apiary <https://my.robojackets.org/docs/officers/payments/accept/#recording-an-offline-payment>`_ at the time the payment is accepted from the student, and Apiary will send an email receipt to the student.

Reviewer: ```suggestion
Payments must be `recorded in Apiary <https://my.robojackets.org/docs/officers/payments/accept/#recording-an-offline-payment>`_ at the time the payment is accepted from the student. Apiary will send an email receipt to the student.
```

Cash should be kept in the safe when not in transit, and must be `deposited to our custodial account <https://sofo.gatech.edu/procedures/depositing-funds>`_ by a student officer as soon as practical.

`Mercury doesn't accept cash deposits <https://support.mercury.com/hc/en-us/articles/28778896984596-Using-your-Mercury-debit-card-at-ATMs>`_.

> Because cash isn't held in a corporate account, we don't track undeposited cash or custodial account balances on corporate books.

Reviewer: ```suggestion
Because cash isn't held in a corporate account, we don't track it on corporate books.
```
